src/libawkward/array/NumpyArray.cpp

```diff
@@ -3424,10 +3424,6 @@ namespace awkward {
                            bool ascending,
                            bool stable,
                            bool keepdims) const {
-    if (length() == 0 ) {
-      return shallow_copy();
-    }
-
```

Comment:
@jpivarski - if performance is a concern, this could be changed to return an `"l"` dtype NumpyArray such as:
```python
<NumpyArray format="l" shape="0" data="" at="0x000000000000"/>
```

Reply:
It's not a concern. Thanks for fixing this!

Reply:
Thanks for checking!